dev/pyproject.py

```diff
@@ -102,6 +102,7 @@ def build(skinny: bool) -> None:
                 "sqlserver": ["mlflow-dbstore"],
                 "aliyun-oss": ["aliyunstoreplugin"],
                 "xethub": ["mlflow-xethub"],
+                "jfrog":["mlflow-jfrog-plugin"]
```

Comment:
i think this just needs a comma after it

Reply:
also added a space after the colon